pkg/pillar/build-rstats.yml

```diff
@@ -16,7 +16,6 @@ config:
     - /:/hostfs
```

Comment:
Although it's not mandatory, I do like when commits mark in the commit message the module they are touching by using `<module> : <commit message>` template. In this commit you just said "Remove net from build yaml", but there are a lot of build.yml files throughout the repository. Using something like `pillar: Remove net from build-rstats.yml file` would let the message more clear and easy to get the goal when using `git log --oneline`, for instance.